lib/videoClient.ts

```diff
@@ -70,7 +70,7 @@ const zoomAuth = (credential) => {
           },
         });
         credential.key.access_token = responseBody.access_token;
-        credential.key.expires_in = Math.round(+new Date() / 1000 + responseBody.expires_in);
+        credential.key.expires_in = Math.round(+new Date() / 1000 + responseBody.expires_in) * 1000;
```

Comment:
What does this change do, as I'm not familiar with how the video integrations work?

Reply:
It changes the credential.key.expires to milliseconds instead of seconds. It could be just me, but the token kept reading as expired for me when credential.key.expires_in was in seconds, and I couldn't use Zoom.